.github/CONTRIBUTING.md

```diff
@@ -53,9 +53,9 @@ make testacc TEST=./google TESTARGS='-run=TestAccContainerNodePool_basic'
 
 The `TESTARGS` variable is regexp-like, so multiple tests can be run in parallel by specifying a common substring of those tests (for example, `TestAccContainerNodePool` to run all node pool tests). There are 1500+ tests, and running all of them takes over 4 hours and requires a lot of GCP quota.
 
-### Ensuring no plan-time difference to upstream provider
+### Ensuring no plan-time difference to published provider
```

Comment:
```suggestion
### Ensuring no plan-time difference to published provider (optional)
```

optional nit: I've seen some pretty literal reads of this page from contributors, we can make it a little more clear that we don't require contributors to perform this level of testing generally

Reply:
done